src/utils/read.js

```diff
@@ -7,16 +7,10 @@ const fs = require("fs");
  * var read = require('./helpers/fsExists')({});
  * read('.babelrc'); // file contents...
  */
-module.exports = function(cache) {
-  cache = cache || {};
```

Comment:
@danez now cache provide old config file when changes happens, i known what it is not best, I'll try to find a solution, also related to `resolve-rc.js`. While it's possible that caching such a place is just the wrong solution.